### DISCUSSION TITLE
[Live Updates] Known ongoing incident with wallets using WalletConnect on Celo (Resolved)

### DISCUSSION THREAD

**Discussion by nicolasbrugneaux** [celo-org/celo-monorepo#10412] [Q&A] [answered]
Hi everyone 🌱 

## TLDR

> **Note**
> + A temporary issue is affecting all Wallets using the WalletConnect protocol in order to connect to `safe.celo.org` and `stcelo.xyz`.
> + The issue relates to **connecting** with these two services with wallets using the WalletConnect protocol. This includes but isn't limited to: `Valora`, `Omni`, `Rainbow`, `TrustWallet`, `CeloTerminal`, `CeloWallet`, etc.. As such it does not affect other wallets such as `Metamask`, `Ledger`, `Brave`, etc.
> + During the incident, we recommend you use Metamask or Ledger to connect to these two websites.

We apologise for the inconvenience and are working hard to bring the service back. Thank you for your patience! 🙏 

## Live updates

Status: Resolved 🟢 
- https://github.com/celo-org/staked-celo-web-app/discussions/138

Status: Resolved 🟢 
- https://github.com/celo-org/web-core/discussions/20

## Have you seen similar bugs?

👉 Please file a bug report in a [Github Discussion in this repository](https://github.com/celo-org/celo-monorepo/discussions).

This will help us debug efficiently, collect similar bug reports, and allow us to refer back to it in the future if someone else runs into the same problem.

Thank you!

**Comment by nicolasbrugneaux** [comment]
> ## Update Tuesday, July 4th (evening CET)
> 
> Hello again,
> 
> the issue is now **mitigated** through a [temporary fix](https://github.com/celo-org/staked-celo-web-app/pull/140).
> 
> It means the incident is resolved but we had to compromise on performances. It seems either rainbowkit, wagmi, or walletconnect have an issue when minified and optimized through webpack. We disabled these optimizations in order to re-establish WalletConnect capacities. We'll keep investigating to re-enable thee optimizations in order to have a better UX.
> 
> Thank you all for your understanding
> 
> _Originally posted by @nicolasbrugneaux in https://github.com/celo-org/staked-celo-web-app/discussions/138#discussioncomment-6355756_

**Comment by nicolasbrugneaux** [comment, marked as answer]
> ## Update Thursday, July 6th
> 
> Hello again,
> 
> the issue is now resolved with minification re-enabled. It was due to a bug in the [SWC](https://swc.rs/) minifier. Using Babel's minifier instead solves the issue completely.
> 
> _Originally posted by @nicolasbrugneaux in https://github.com/celo-org/staked-celo-web-app/discussions/138#discussioncomment-6371472_
